src/pages/saksbehandling/sakintro/Sakintro.tsx

```diff
@@ -115,6 +118,15 @@ const Sakintro = () => {
                             {intl.formatMessage({ id: 'link.kontrollsamtale' })}
                         </LinkAsButton>
                     )}
+                    {skattemeldingToggle && (
+                        <Button
+                            variant="secondary"
+                            onClick={() => hentSkatt({ fnr: props.søker.fnr })}
+                            loading={RemoteData.isPending(skattemelding)}
+                        >
+                            {intl.formatMessage({ id: 'skattemelding.hent' })}
+                        </Button>
+                    )}
```

Comment:
Er dette for knappen som vises i toppen av saken? I så fall kan vel den bare fjernes?

Reply:
Ja, fjerner. 👍